nullaway/src/main/java/com/uber/nullaway/ErrorProneCLIFlagsConfig.java

```diff
@@ -108,6 +113,11 @@
       throw new IllegalStateException(
           "Invalid -XepOpt" + FL_SUPPRESS_COMMENT + " value. Comment must be single line.");
     }
+    /** --- JarInfer configs --- */
+    jarInferUseReturnAnnotations = flags.getBoolean(FL_JI_USE_RETURN).orElse(false);
+    jarInferRegexStripModelJarName =
```

Comment:
Going to edit these to document the default. Other than that, just remind me to test performance before we land the internal diff.